lib/SILGen/SILGen.cpp

```diff
@@ -1485,8 +1483,7 @@ SILModule::constructSIL(ModuleDecl *mod, SILOptions &options, FileUnit *SF,
 }
 
 std::unique_ptr<SILModule>
-swift::performSILGeneration(ModuleDecl *mod,
-                            SILOptions &options,
+swift::performSILGeneration(ModuleDecl *mod, SILOptions &options,
```

Comment:
In the future, please try to avoid changes that are just reformatting in a file you aren't otherwise editing.

Reply:
So I checked, and these changes were made by git-clang-format. I understand that they make it harder to review code, but also understand that the uniformity obtained by git-clang-format can make it easier. I had thought that the best practice was to use git-clang-format, but from your comment I'm surmising that best practice would be to use git-clang-format, then undo any changes that it made and still stay within 80-characters. Is that correct?

Concerning this particular file, I had changed it and then changed it back. Should have then undone the git-clang-formatting changes. But let me know which practice would serve the team best in the future. Thanks!

For this PR, I will go through all my changes and undo all similar ones.

Reply:
The best practice is to only format the areas of code that you touch, not the whole file, but having the rest of the file get cleaned up is all right as well. But yes, that last part is the important part: if you end up *not* changing a file, please remove any formatting changes as well; we can do them some other time.

Reply:
OK, thanks a lot for clarifying! I may not be running git-clang-format in the optimum fashion. I think I've now caught all of those.